### DISCUSSION TITLE
Welcome to dtails Discussions!

### DISCUSSION THREAD

**Discussion by DesobedienteTecnologico** [DesobedienteTecnologico/dtails#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ivanwmco** [comment]
I stumbled across TailsOS & dtails while researching Bitcoin wallets.  I have a few questions:
- Are the dtails applications installed in TailsOS 'persistant storage'?
- Does dtails have minimal hardware requirements (USB pen drive capacity, etc.)?
- Why does the Specter Desktop installation stall on  'NameError: name 'print_green' is not defined''?

**Reply by DesobedienteTecnologico** [reply]
Hello,

1. It do not enable persistant storage by default. You have to enable it and type your own password.
2. Min. 3GB (All depends on the software you install)
3. It's a bug. I think is already solved. That variable is used to print text in green color.

**Comment by ivanwmco** [comment]
While trying to use dtails to install all applications (except Specter Desktop and Whirlpool GUI) SimpleX Desktop could not be configured because of missing dependency 'libssl1.1'. Turns out that Ubuntu 24.04 LTS comes with 'libssl3' and 'libssl1.1'  is considered obsolete and no longer available in Ubuntu 24.04 LTS packages.

**Reply by DesobedienteTecnologico** [reply]
Do you still have this problem?

**Comment by ivanwmco** [comment]
Another issue arose while testing:
Feather Wallet AppImage could not be found, probably because of a new version...

**Reply by DesobedienteTecnologico** [reply]
That issue is rare because it selects the version specified in the variable. It doesn't really matter that a new version is rolled out. Do you still have this issue?

**Comment by HUX13Y** [comment]
Hey man, why can I not close the git? That command isnt working for me on Linux Mint...
